.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea
 nbproject
-composer.lock
\ No newline at end of file
+composer.lock
+vendor
```

Comment:
There is no need for ignoring vendor directory because this package is supposed to be used in Yii2 application. Therefore the package will be in the vendor dir of Yii2 application.